luminoth/tools/server/web.py

```diff
@@ -89,7 +70,11 @@ def web(config_files, host, port, debug):
         # Gets the names of the classes
         classes_file = os.path.join(config.dataset.dir, 'classes.json')
         if tf.gfile.Exists(classes_file):
-            app.config['class_labels'] = json.load(
+            config['class_labels'] = json.load(
                 tf.gfile.GFile(classes_file))
 
+    # Initialize model
+    global PREDICTOR_NETWORK
+    PREDICTOR_NETWORK = PredictorNetwork(config_files)
```

Comment:
Do this mean that when running `lumi server web` you can't access the website until the network is fully loaded?

Reply:
Yep. Also separating prediction and model creation will make it cleaner to support switching between different models when we add SSD, Retina, etc.